src/app/features/shows/pages/movie-details/movie-details.component.html

```diff
@@ -1 +1,44 @@
-<p>movie-details works!</p>
+<div *ngIf="loading; else yes">
```

Comment:
```suggestion
<div *ngIf="loading; else showMovieDetails">
```
đặt tên rõ ràng ra e